packages/riipen-ui-docs/src/modules/components/MarkdownPage.jsx

```diff
@@ -11,141 +18,174 @@ import Menu from "src/modules/components/Menu";
+          </Popover>
+        </div>
+      </AppBar>
+      <div className="page">
+        <div className="menu">
+          <Menu />
+        </div>
+        <div className="content">
+          {sections.map((section, index) => {
+            if (demos && constants.REGEX.DEMO.test(section)) {
+              let options;
+
+              try {
+                options = JSON.parse(`{${section}}`);
+              } catch (err) {
+                console.error(err);
+                return null;
+              }
+
+              const name = options.demo;
+
+              if (!demos || !demos[name]) {
+                throw new Error(
+                  [
+                    `Missing demo: ${name}. You can use one of the following:`,
+                    Object.keys(demos)
+                  ].join("\n")
                 );
               }
 
-              return <MarkdownElement key={index} text={section} />;
-            })}
-            <div className="footer">
-              <Divider />
-              <Footer />
-            </div>
-          </div>
-          <div className="toc">
-            <TOC sections={sections} />
+              return <Demo key={index} demo={demos[name]} options={options} />;
+            }
+
+            return <MarkdownElement key={index} text={section} />;
+          })}
+          <div className="footer">
+            <Divider />
+            <Footer />
           </div>
         </div>
-        <style jsx>{`
-          .page {
-            display: flex;
-            justify-content: space-between;
-            padding-top: 22px;
-            position: relative;
-            top: 50px;
-            width: 100%;
-          }
-          .page :global(*:target) {
-            padding-top: 50px;
-          }
-          .content {
-            padding: 0 20px;
-            overflow-x: auto;
-            width: 100%;
-          }
-          .menu {
-            padding: ${theme.spacing(2)}px;
-          }
+        <div className="toc">
+          <TOC sections={sections} />
+        </div>
+      </div>
+      <style jsx>{`
+        :global(.appBar) {
+          justify-content: space-between;
+        }
+        .page {
+          display: flex;
+          justify-content: space-between;
+          padding-top: 22px;
+          position: relative;
+          top: 50px;
+          width: 100%;
+        }
+        .page :global(*:target) {
+          padding-top: 50px;
+        }
+        .content {
+          padding: 0 20px;
+          overflow-x: auto;
+          width: 100%;
+        }
+        .menu {
+          padding: ${theme.spacing(2)}px;
+        }
+        .menuMobile {
+          display: none;
+        }
+        :global(.menuPopover) {
+          margin: ${theme.spacing(3)}px ${theme.spacing(2)}px;
+          max-height: calc(100% - ${theme.spacing(13)}px) !important;
+          max-width: calc(100% - ${theme.spacing(4)}px) !important;
```

Comment:
Some overriding of popover.  Should I use linkedStyles (css.resolve) instead? I was lazy and just used importants